src/macaron/slsa_analyzer/build_tool/gradle.py

```diff
@@ -217,13 +230,17 @@ def get_group_id(self, gradle_exec: str, project_path: str) -> str | None:
             The group id of the project, if exists.
         """
         try:
+            logger.debug(
```

Comment:
Is this meant to be an `INFO` level log ?

Reply:
See commit [fd880ec](https://github.com/oracle/macaron/pull/446/commits/fd880ec99dc041209783be5fc8980d7f0163966c)